src/MapPane.js

```diff
@@ -0,0 +1,158 @@
+/* @flow */
+import React, { PropTypes, Component } from 'react'
+import { forEach, omit, uniqueId, indexOf } from 'lodash'
+
+import childrenType from './types/children'
+import mapType from './types/map'
+import paneType from './types/pane'
+
+const BLACKLIST = ['tile', 'shadow', 'overlay', 'map', 'marker', 'tooltip', 'popup']
+
+export default class MapPane extends Component {
+  static propTypes = {
+    name: PropTypes.string,
+    children: childrenType,
+    map: mapType,
+    zIndex: PropTypes.number,
```

Comment:
Syntactic sugar. Since the main purpose of leaflet panes is to have more control over of zIndexes I included it as a prop. Let me know if you want this dropped.